script.js

```diff
@@ -1,4 +1,150 @@
+// Roll dice and put them in order and make highest number possible
+// User will roll dices - include dice function
+// Arrange numbers - string
+// User win if their number  is bigger
+// user lose if their number is smaller
+
+// Need to add while to loop entire function for player 1,2,3.
+
+// Variables
+var myOutputValue = "";
+var currentGameMode = "player 1";
+
+// Dice Roll
+var diceRoll = function () {
+  var randomDecimal = Math.random() * 6;
+  var randomInteger = Math.floor(randomDecimal);
+  var diceNumber = randomInteger + 1;
+  return diceNumber;
+};
+
+//Diceroll Input (1)
+var player1Dice1 = diceRoll();
+console.log(`Player 1 Dice 1 rolled ${player1Dice1}`);
+
+var player1Dice2 = diceRoll();
+console.log(`Player 1 Dice 2 rolled ${player1Dice2}`);
+
+//Diceroll Input (2)
+var player2Dice1 = diceRoll();
+console.log(`Player 2 Dice 1 rolled ${player2Dice1}`);
+
+var player2Dice2 = diceRoll();
+console.log(`Player 2 Dice 2 rolled ${player2Dice2}`);
+
+//Player 1 functions
+var player1Roll = function () {
+  var player1Response = `Player 1 rolled ${player1Dice1} for dice 1 and ${player1Dice2} for dice 2. <br><br> Choose the order of the dice.`;
+  return player1Response;
+};
+
+var player1Choose1 = function () {
+  var choose1First = `${player1Dice1}${player1Dice2}`;
+  var dice1Chosen = `Player 1, you choose Dice 1 first. <br> Your number is ${choose1First}. <br><br> It is now Player 2's turn. <br> Press Submit to continue!`;
+  return dice1Chosen;
+};
+
+var player1Choose2 = function () {
+  var choose2First = `${player1Dice2}${player1Dice1}`;
+  var dice2Chosen = `Player 1, you choose Dice 2 first. <br> Your number is ${choose2First}. <br><br> It is now Player 2's turn. <br> Please press submit to continue!`;
+  return dice2Chosen;
+};
+
+//Player 2 functions
+var player2Roll = function () {
+  var player2Response = `Player 2 rolled ${player2Dice1} for dice 1 and ${player2Dice2} for dice 2. <br><br> Choose the order of the dice.`;
+  return player2Response;
+};
+
+var player2Choose1 = function () {
+  var choose1First2 = `${player2Dice1}${player2Dice2}`;
+  var dice1Chosen2 = `Player 2, you choose Dice 1 first. <br> Your number is ${choose1First2}. <br> Lets tabulate the scores. <br><br>Press submit to find out who won!`;
+  return dice1Chosen2;
+};
+
+var player2Choose2 = function () {
+  var choose2First2 = `${player2Dice2}${player2Dice1}`;
+  var dice2Chosen2 = `Player 2, you choose Dice 2 first. <br> Your number is ${choose2First2}. <br> Lets tabulate the scores. <br><br>Press submit again to find out who won!`;
+  return dice2Chosen2;
+};
+
```

Comment:
Clear and distinct functions are good for readability but since you are repeating them, maybe you can consider writing functions with arguments so you can reuse a single function with the different arguments like 

var playerChoose = function (player , choice ) {
  if (choice == 1 ) {
      if ( player == 1 ) { .........}}}